src/ssl/ssl_rec_gcm.c

```diff
@@ -42,6 +42,10 @@ gen_gcm_init(br_sslrec_gcm_context *cc,
 	bc_impl->init(&cc->bc.vtable, key, key_len);
 	cc->gh = gh_impl;
 	memcpy(cc->iv, iv, sizeof cc->iv);
+	if (key_len == 16)
+		memcpy(cc->key.k16, key, sizeof cc->key.k16);
```

Comment:
This if can be omitted, and you can write just memcpy(cc->key.k32, key, key_len); since k16 and k32 share the same
memory location.

Reply:
Fixed, thanks!